xml/System.ComponentModel.DataAnnotations/IValidatableObject.xml

```diff
@@ -25,7 +25,7 @@
   <Interfaces />
   <Docs>
     <summary>Provides a way for an object to be invalidated.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>This allows to check if an instance that implements the interface is valid or not.</remarks>
```

Comment:
```suggestion
    <remarks>Instances of types that implement this interface can be checked to determine if they are valid.</remarks>
```

Reply:
This looks good to me except (as a really tiny comment) I would spell out "they are". I would also make the change to the summary above.